apps/fluent-tester/src/FluentTester/TestComponents/Shadow/ShadowButtonTestSection.tsx

```diff
@@ -3,8 +3,10 @@ import { Platform, View } from 'react-native';
 import { FAB, Text } from '@fluentui/react-native';
 import { commonTestStyles, stackStyle } from '../Common/styles';
 
-const CustomFABNoShadow = FAB.customize({ shadowDepth: undefined });
-const CustomFABShadow64 = FAB.customize({ shadowDepth: 'shadow64' });
+const CustomFABNoShadow = FAB.customize({ shadowToken: undefined });
+const CustomFABShadow64 = FAB.customize({
+  shadowToken: { ambient: { x: 0, y: 0, blur: 8, color: '#00000033' }, key: { x: 0, y: 32, blur: 64, color: '#0000003d' } },
+});
```

Comment:
Wasn't sure exactly how to grab the shadow from the theme here, since the useTheme() hook needs to be called from a function component, so I just manually copied it in.